### ISSUE TITLE
[ON HOLD - Waiting on #26190] Dev - User is unable to unlink secondary contact method

### ISSUE THREAD

**Issue by kavimuru** [Expensify/App#25472]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

## Action Performed:
1. Add a secondary contact method email address B & leave it unverified
2. Open a new window
3. Navigate to a sign-up page
4. Enter email address B
5. Click on the Continue button
## Expected Result: 
User should navigate to unlink page where they can proceed to unlink the secondary contact method
## Actual Result: 
User is not navigated to the unlink page
## Workaround:
Can the user still use Expensify without this being fixed? Have you informed them of the workaround?

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [ ] Android / native
- [ ] Android / Chrome
- [ ] iOS / native
- [ ] iOS / Safari
- [x] MacOS / Chrome / Safari
- [ ] MacOS / Desktop

**Version Number:** DEV
**Reproducible in staging?:** dev
**Reproducible in production?:** dev
**If this was caught during regression testing, add the test name, ID and link from TestRail:**
**Email or phone of affected tester (no customers):**
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Notes/Photos/Videos:** Any additional supporting documentation

https://github.com/Expensify/App/assets/43996225/fe13ba78-e58c-48d3-a27f-498242a7eb7d

**Expensify/Expensify Issue URL:**
**Issue reported by:** @Natnael-guchima
**Slack conversation:** https://expensify.slack.com/archives/C049HHMV9SM/p1692340650184629

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by Julesssss** [comment]
@Beamanator is this occurring on prod, or just staging?

**Comment by Julesssss** [comment]
As this was reported just 6 hours ago on dev, I feel like this should block the NEXT deployment and not `1.3.55-4`. 

If this isn't reproducible on staging/prod I'll check it off

**Comment by Beamanator** [comment]
Ya that makes sense to me 👍

**Comment by johncschuster** [comment]
@Beamanator how's this one going?

**Comment by Natnael-Guchima** [comment]
Last time I checked this was working fine. I can check it again and get back here.

**Comment by Beamanator** [comment]
Thanks @Natnael-Guchima , that would be very helpful 🙏

**Comment by Natnael-Guchima** [comment]
I tested it 3 times, and it worked once and failed two times. And In the instance it worked, system doesn't navigate from 'unlink' page to 'sign-in' page. Similar issue happens when authenticating with magic link, [Link ](https://github.com/Expensify/App/issues/25979)to similar issue where authenticating with magic link is not working. 

https://github.com/Expensify/App/assets/57330233/d1ae57a7-bf4c-4e65-9a0c-72fe0529341b

https://github.com/Expensify/App/assets/57330233/34d1ce13-a017-4521-bf31-56627bb5b7fd

**Comment by Beamanator** [comment]
Thanks @Natnael-Guchima ! I will try to investigate tomorrow if this needs to be made internal or external

**Comment by Natnael-Guchima** [comment]
You are welcome 🙌

**Comment by Beamanator** [comment]
Looks like this is now being worked on here: https://github.com/Expensify/App/issues/26190 (the other linked issue was closed as a dupe)

@Natnael-Guchima do you think the linked issue will fix this too? We can put this on hold in the meantime

**Comment by Natnael-Guchima** [comment]
@Beamanator it might solve the issue. We can put this on hold, and see if the other can fix this too.

**Comment by johncschuster** [comment]
Thanks! I've put this one on hond and will downgrade it to a Weekly priority

**Comment by johncschuster** [comment]
Still on hold

**Comment by Beamanator** [comment]
still on hold

**Comment by johncschuster** [comment]
Still on hold

**Comment by Natnael-Guchima** [comment]
I will test this to see if this is working fine since the holding pr is merged

**Comment by Natnael-Guchima** [comment]
@johncschuster this working fine now. 

<img width="1352" alt="Screenshot 2023-10-06 at 7 11 38 AM" src="https://github.com/Expensify/App/assets/57330233/63653dab-22e4-4d07-bec4-1e5f5b3b3c3a">

**Comment by johncschuster** [comment]
Great! Thanks for confirming that, @Natnael-Guchima! In that case, I'll go ahead and close the issue.
